src/rest-api/class-wp-rest-import-attachments.php

```diff
@@ -0,0 +1,91 @@
+<?php
+
+class WP_REST_Import_Attachments {
+	const NAMESPACE = 'wordpress-importer/v1';
```

Comment:
👍 (https://developer.wordpress.org/rest-api/extending-the-rest-api/routes-and-endpoints/#namespaces)

Reply:
Maybe `wp-importer`, stemming from `wp` namespace for Core?

Reply:
@creativecoder -- I started w/ `wp-importer` but decided to it the same as the plugin name for the first pass.  Happy to change the namespace to whatever, though :)